setup.py

```diff
@@ -24,6 +24,7 @@
         "sphinx_rtd_theme",
         "flynt",
     ],
+    "readthedocs": ["sphinx<3.0"],
```

Comment:
Won't it need `sphinx-rtd-theme` as well?

Reply:
That should already exist on rtd, but we'll see.

Reply:
Yes, it works without specifying that.